tests/integration/base.py

```diff
@@ -43,3 +43,13 @@ class SparklyTestSession(SparklySession):
         'collect_max': 'brickhouse.udf.collect.CollectMaxUDAF',
         'length_of_text': (lambda text: len(text), StringType())
     }
+
+
+class SparklyTestSessionWithES7(SparklySession):
```

Comment:
Should we have the global session use ES7, and have the special, one-off session use ES6? The reason is that going forward, ES7 will be the way to go, and eventually support for ES6 will get deprecated